admin/classes/domain/Resource.php

```diff
@@ -49,16 +51,20 @@ public function getResourceByTitle($title){
 		return $objects;
 	}
 
+  //returns resource objects by title
+	public function getResourceByIsbnOrISSN($isbnOrISSN){
 
+		$query = "SELECT DISTINCT(resourceID)
+			FROM IsbnOrIssn";
 
-	//returns array of related resource objects
-	public function getParentResource(){
+    $i = 0;
+    foreach ($isbnOrISSN as $value) {
+      $query .= ($i == 0) ? " WHERE " : " OR ";
+      $query .= "isbnOrIssn = '$value'";
```

Comment:
When including parameters in SQL queries, we should sanitize the input using something like mysqli_real_escape_string() or CORAL's escapeString() function. We should also test that $isbnOrISSN is an array.